win32/src/win32apimodule.cpp

```diff
@@ -6319,6 +6309,7 @@ PYWIN_MODULE_INIT_FUNC(win32api)
         pfnRegOpenCurrentUser = (RegOpenCurrentUserfunc)GetProcAddress(hmodule, "RegOpenCurrentUser");
         pfnRegOverridePredefKey = (RegOverridePredefKeyfunc)GetProcAddress(hmodule, "RegOverridePredefKey");
     }
+    // clang-format on
```

Comment:
I think those format comments are also redundant (I'm guessing another leftover?). Running clang-format locally I'm not seeing any difference

 ```suggestion
```